tests/autoria/test_average.py

```diff
@@ -0,0 +1,51 @@
+import requests_mock
+import json
+
+from autoria.api import RiaAverageCarPrice
+
+
+class TestAverage:
+    """Tests for autoria average car price calculation."""
+
+    def test_average_minimal(self, ria_categories, ria_marks, ria_states,
```

Comment:
Have I understood that "minimal" test (test_average_minimal) is intended to run test on limited set of parameters? I was up to use all parameters that we have in the test, but suddenly noticed "minimal" in the name and now I'm hesitating should I create one more test or I can expand functionality of this one?